src/decoder/lattice-incremental-decoder.cc

```diff
@@ -1110,6 +1127,8 @@ bool LatticeIncrementalDecoderTpl<FST, Token>::GetIncrementalRawLattice(
   // step 1.3 create final_arc for later appending with the next chunk
   if (create_final_state) {
     StateId end_state = ofst->AddState(); // final-state for the chunk
+    // the extra_cost and alpha_cost of initial state are 0 and best_cost
+    backward_costs->push_back(0); 
```

Comment:
I'm not convinced that this, and the backward-cost of the initial state, are quite right, because their alphas may not be super-close to zero.  Is there a way to work out a reasonable alpha for the initial state and final state and work out the best cost from that, based on extra_cost = 0?  This will be not super far from zero because of the renormalization we do on each frame, but may still make a small difference.  This *should* have worked.  I'm not sure why it's not helping.